dev-tools/packer/readme.md.j2

```diff
@@ -0,0 +1,23 @@
+# Welcome to {{.beat_name}} {{.version}}
+
+{{.beat_description}}
+
+## Getting Started
+
+To get running with your beat, setup elasticsearch on your localhost and start it with:
```

Comment:
..."set up Elasticsearch" ("setup" should be "set up" when you're using it as a verb). :-)